dbms/src/test/java/org/polypheny/db/jdbc/JdbcAutoCommitTest.java

```diff
@@ -117,7 +117,7 @@ public void testDml() throws SQLException {
 
                     }
                 } finally {
-                    statement.executeUpdate( "DROP TABLE IF EXISTS autocomit" );
+                    statement.executeUpdate( "DROP TABLE IF EXISTS autoCommit" );
```

Comment:
There is, in fact, a typo (it misses an "m" in commit). However, the casing is valid since relational namespaces are case-insensitive per default. Is there a particular reason for adjusting the casing in the Drop DDL?

Reply:
Yes, it is spelled with a capital C everywhere else in the file, just making it consistent.

Reply:
Yes, but it should also work with different casing. While this test should check auto-commit functionality, it also checks support for different casings (which I agree is not a good test design). So adjusting this makes sense (especially since it is the same casing in the drop clause of both tests). However, I think we should make sure to add a dedicated set of tests for checking case (in)sensitivity in future.